### ISSUE TITLE
🆕 Software Suggestion | git-ssb, a blockchain-based decentralized git repositories tracker

### ISSUE THREAD

**Issue by lrq3000** [privacytoolsIO/privacytools.io#1685]
## Basic Information

**Name:** git-ssb
**Category:** Productivity Tools
**URL:**
* description: https://github.com/noffle/git-ssb-intro/blob/master/README.md
* sourcecode: https://git.scuttlebot.io/%25n92DiQh7ietE%2BR%2BX%2FI403LQoyf2DtR3WQfCkDKlheQU%3D.sha256
* tracker: https://git.scuttlebot.io/

## Description
git-ssb is a decentralized local git repositories manager and tracker. It provides a webapp interface similar to GitHub, and published repositories are accessible globally on the tracker, which is managed on a secure scuttlebutt chain (similar transactional nature to a blockchain but without money).

## Why I am making the suggestion

Until the [forgefed](https://notabug.org/peers/forgefed), this is the only decentralized GitHub-like solution that also provides a form of federation currently, in the form of a global tracker of all repositories. But as I understand it, it's not true federation but rather more like a blockchain, since each repository still stays locally hosted on the user machine (which is both a good point for privacy and a bad point for availability - but I may have misunderstood something here, maybe other nodes help in availability when your machine is offline).

In other words and to make a simple global picture of how it works: you make a local git repository on your computer, then publish via git-ssb on localhost, and it gets announced in the ssb-git tracker globally so that any other user can access your repository and clone/fork/pull request on it. So you get the best of both worlds: full ownership of your repository, but still get a global announcement in a shared tracker.

## My connection with the software

None. I would just like to see federated/decentralized git trackers become a reality :-)

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by lrq3000** [comment]
This could also maybe solve the censorship issue raised in #1062, not necessarily for PTIO (I'm not convinced it should move to this protocol), but for individuals in embargo'ed countries, as I can't see how such repos could be blocked since the individuals are in full control of the repos. The only practical scenario would be for the tracker nodes to all agree on a blacklist to block the listing of repos coming from some countries, which is more difficult than for a centralized service (ie, seems impractical.

Also note that git-ssb offers [issues to comment on projects](https://git.scuttlebot.io/%25n92DiQh7ietE%2BR%2BX%2FI403LQoyf2DtR3WQfCkDKlheQU%3D.sha256/issues), although they are readable through the clearnet, creating and commenting on issues is not open on the clearnet (or I did not find how to do that), I think you need to install and connect through ssb (secure scuttlebutt).
